src/System.IO.FileSystem.Watcher/src/System/IO/FileSystemWatcher.OSX.cs

```diff
@@ -155,12 +159,16 @@ private void CreateStreamAndStartWatcher()
                 // Create the callback for the EventStream
                 _callback = new Interop.EventStream.FSEventStreamCallback(FileSystemEventCallback);
 
+                // Make sure the OS file buffer(s) are fully flushed so we don't get events from cached I/O
+                // like we saw in the Unit Tests
```

Comment:
Comments like "like we saw in the Unit Tests" get stale pretty quickly... someone reading this in the not-too-distant-future won't know what that's referring to.  I'd suggest just cutting that phrase; the previous part of the comment does a good job explaining the reasoning.